documentation/docs/mock-apps/components/qr-code-scanner-dialog-button.md

```diff
@@ -23,6 +23,31 @@ The QRCodeScannerDialogButton component is responsible for rendering a button th
 }
 ```
 
+## Diagram
+
+```mermaid
+sequenceDiagram
+    participant U as User
+    participant MA as Mock App
+    participant V as VCKit
+
+    U->>MA: Scan QRCode
+    MA->>MA: Fetch data from QRCode URL
```

Comment:
@namhoang1604, what are the permitted data structures of the QRpayload? Can we also use a Data Matrix? What are the permitted data structures of the file being fetched? What are the allowed file extensions? What are the permitted credential types? What happens if the contents of the credential being fetched are encrypted?

Reply:
The data structure is defined [here](https://github.com/uncefact/tests-untp/blob/1fc228a6b367a70b4e1c5a07b2dc4707f30c78ae/documentation/docs/mock-apps/components/qr-code-scanner-dialog-button.md#props), it allows VC or JSON. The current implementation is not handling the encrypted data.

Reply:
Thanks @ashleythedeveloper, I updated it for more details.